web/satellite/src/components/account/billing/billingTabs/Overview.vue

```diff
@@ -1,22 +1,266 @@
+        let currentYear = rawDate.getFullYear();
+        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
+  "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+        this.currentDate = `${monthNames[rawDate.getMonth()]} ${currentYear}`
+    }
+
+    /**
+     * Returns account balance from store.
+     */
+    public get balance(): AccountBalance {
+        return this.$store.state.paymentsModule.balance;
+    }
+
+    /**
+     * projectUsageAndCharges is an array of all stored ProjectUsageAndCharges.
+     */
+    public get projectUsageAndCharges(): ProjectUsageAndCharges[] {
+        return this.$store.state.paymentsModule.usageAndCharges;
+    }
+
+    /**
+     * priceSummary returns price summary of usages for all the projects.
+     */
+    public get priceSummary(): number {
+        return this.$store.state.paymentsModule.priceSummary;
+    }
+
+    public routeToBillingHistory(): void {
+        this.$router.push(RouteConfig.Account.with(RouteConfig.Billing).with(RouteConfig.BillingHistory2).path);
+    }
+
+    public routeToPaymentMethods(): void {
+        this.$router.push(RouteConfig.Account.with(RouteConfig.Billing).with(RouteConfig.BillingPaymentMethods).path);
+    }
 
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped lang="scss">
+    .total-cost {
+        font-family: sans-serif;
+        margin: 20px 0;
+        &__header-container {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            &__date {
+                display: flex;
+                justify-content: space-between;
+                align-items: bottom;
+                color: #56606d;
+                font-weight: 700;
+                font-family: sans-serif;
+                border: 1px solid #D8DEE3;
+                border-radius: 5px;
+                background-color: #fff;
+                height: 15px;
+                width: auto;
+                padding: 10px;
+                box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.08);
+            }
+        }
+
+        &__card-container {
+            display: flex;
+            justify-content: space-between;
+            flex-wrap: wrap;
+        }
+
+        &__card {
+            width: calc(50% - 50px);
+            min-width: 188px;
+            box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.04);
+            border-radius: 10px;
+            background-color: #fff;
+            padding: 20px;
+            margin-top: 20px;
+            display: flex;
+            flex-direction: column;
+            justify-content: left;
+            position: relative;
+
+            &__money-text {
+                font-weight: 800;
+                font-size: 32px;
+                margin-top: 10px;
+            }
+
+            &__label-text {
+                font-weight: 400;
+                margin-top: 10px;
+                min-width: 200px;
+            }
+
+            &__link-text {
+                font-weight: 500;
```

Comment:
not sure but instead of this weight you should use medium/semiBold font

Reply:
changed to medium